Modules/Utils/CLI/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
This should be outside Utils

Reply:
No, it shouldn't; these are *utilities* that can always be used by passsing a fake cli object (depends on the function)